submit-api/src/submit_api/services/package.py

```diff
@@ -304,8 +306,21 @@ def submit_package(cls, package_id):
         with session_scope() as session:
             package = cls._get_and_validate_complete_package(package_id)
             if package.submitted_on:
-                return cls._resubmit_package(package, session)
-            return cls._submit_package(package, session)
+                submitted_package: PackageModel = cls._resubmit_package(package, session)
+            else:
+                submitted_package: PackageModel = cls._submit_package(package, session)
+
+            ActivityLogService.log_activity(
```

Comment:
better this be in a separate method to avoid cluttering submit_package. Maybe we can create def _submit_package_activity_log